extensions/cornerstone/src/utils/measurementServiceMappings/EllipticalROI.js

```diff
@@ -85,7 +85,7 @@ function getMappedAnnotations(annotation, DisplaySetService) {
   const targets = Object.keys(cachedStats);
 
   if (!targets.length) {
-    return;
+    return [];
```

Comment:
Because we are adding the measurements earlier (i.e. on start), we need better initial values.